### PR TITLE
Adding sql client to common

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Moving SQL client to azure_rm_common to avoid test problems in the future

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME

azure_rm_common
azure_rm_sqlserver
azure_rm_sqldatabase

##### ANSIBLE VERSION
2.6

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py [hunk 1]
```diff
@@ -432,10 +428,10 @@ def create_update_sqldatabase(self):
         self.log("Creating / Updating the SQL Database instance {0}".format(self.name))
 
         try:
-            response = self.mgmt_client.databases.create_or_update(resource_group_name=self.resource_group,
-                                                                   server_name=self.server_name,
-                                                                   database_name=self.name,
-                                                                   parameters=self.parameters)
+            response = self.sql_client.databases.create_or_update(resource_group_name=self.resource_group,
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py:431]: just wondering how this get merged  at first place..

File: lib/ansible/module_utils/azure_rm_common.py [hunk 2]
```diff
@@ -1060,6 +1062,14 @@ def containerservice_client(self):
                                                                      base_url=self._cloud_environment.endpoints.resource_manager)
         return self._containerservice_client
 
+    @property
+    def sql_client(self):
```
Comment (yungezz) [inline lib/ansible/module_utils/azure_rm_common.py:1066]: is it necessary to specify api versoin?

Comment (zikalino) [reply lib/ansible/module_utils/azure_rm_common.py:1066]: no it's not

File: lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py (hunk 1 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py:431]: why?

**Pull request merged**